p3/main.yaml

```
@@ -0,0 +1,35 @@
+sources:
+  flight_level:
+    args:
+      path: "{{CATALOG_DIR}}/flight-level.yaml"
+    description: 'Flight level data'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  AXBT:
+    args:
+      path: "{{CATALOG_DIR}}/axbts.yaml"
+    description: 'AXBT data'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  remote_sensing:
+    args:
+      path: "{{CATALOG_DIR}}/remote-sensing.yaml"
+    description: 'W-band and SFMR data'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  w_band_radar:
+    args:
+      path: "{{CATALOG_DIR}}/w-band-radar.yaml"
+    description: 'W-band radar data'
+    driver: intake.catalog.local.YAMLFileCatalog
+    metadata: {}
+
+  wsra:
+    args:
+      path: "{{CATALOG_DIR}}/WSRA.yaml"
```

```suggestion
      path: "{{CATALOG_DIR}}/wsra.yaml"
```
The committed file is spelled in lower case. Maybe this didn't turn up on your side, if you are working on a case-insensitive system.

---

Fixed in b6b90b0